src/network/networkd-ipv6-proxy-ndp.c

```diff
@@ -18,7 +18,7 @@ static int set_ipv6_proxy_ndp_address_handler(sd_netlink *rtnl, sd_netlink_messa
         assert(link);
 
         r = sd_netlink_message_get_errno(m);
-        if (r < 0 && r != -EEXIST)
+        if (r < 0)
```

Comment:
If `NLM_F_EXCL` is set then `EEXIST` will come. I think we should better add `NLM_F_EXCL` `sd_rtnl_message_new_neigh` like other which is missing now

Reply:
No. When `NTF_PROXY` is set `pneigh_lookup()` is called, and `EEXIST` never come. There are no cases we should ignore.

Reply:
It's not about `NTF_PROXY` only . In general like in _new

Reply:
Not sure for general case... It should depend on the object we want to add or update...

Reply:
In this case `NTF_PROXY` . But need to consider other cases too

Reply:
If it is worth to update, let's adjust the default flags in another later PR.

Reply:
OK lgtm